Makefile

```diff
@@ -132,97 +127,155 @@ fleetctl: .prefix .pre-build .pre-fleetctl
 fleetctl-dev: GO_BUILD_RACE_ENABLED_VAR=true
 fleetctl-dev: fleetctl
 
+.help-short--lint-js:
+	@echo "Run the JavaScript linters"
 lint-js:
 	yarn lint
 
+.help-short--lint-go:
+	@echo "Run the Go linters"
 lint-go:
 	golangci-lint run --exclude-dirs ./node_modules --timeout 15m
 
+.help-short--lint:
+	@echo "Run linters"
+.help-long--lint:
+	@echo "Runs the linters for Go and Javascript code.  If linter type is not specified, all linters will be run."
+.help-usage--lint:
+	@echo "$(TOOL_CMD) lint [linter-type]"	
+.help-extra--lint:
+	@echo "AVAILABLE LINTERS:"
+	@echo "  go   Lint Go files with golangci-lint"
+	@echo "  js   Lint .js, .jsx, .ts and .tsx files with eslint"
+
+ifdef ARG1
+lint: lint-$(ARG1)
+else
 lint: lint-go lint-js
+endif
 
-dump-test-schema:
+.help-short--test-schema:
+	@echo "Update schema.sql from current migrations"
+test-schema:
 	go run ./tools/dbutils ./server/datastore/mysql/schema.sql
-
+dump-test-schema: test-schema
 
 # This is the base command to run Go tests.
 # Wrap this to run tests with presets (see `run-go-tests` and `test-go` targets).
 # PKG_TO_TEST: Go packages to test, e.g. "server/datastore/mysql".  Separate multiple packages with spaces.
 # TESTS_TO_RUN: Name specific tests to run in the specified packages.  Leave blank to run all tests in the specified packages.
 # GO_TEST_EXTRA_FLAGS: Used to specify other arguments to `go test`.
-# GO_TEST_MAKE_FLAGS: Internal var used by other targets to add arguments to `go test`.
-#
-PKG_TO_TEST := "" # default to empty string; can be overridden on command line.
+# GO_TEST_MAKE_FLAGS: Internal var used by other targets to add arguments to `go test`.						 
+PKG_TO_TEST := ""
 go_test_pkg_to_test := $(addprefix ./,$(PKG_TO_TEST)) # set paths for packages to test
 dlv_test_pkg_to_test := $(addprefix github.com/fleetdm/fleet/v4/,$(PKG_TO_TEST)) # set URIs for packages to debug
-
-DEFAULT_PKG_TO_TEST := ./cmd/... ./ee/... ./orbit/pkg/... ./orbit/cmd/orbit ./pkg/... ./server/... ./tools/...
-ifeq ($(CI_TEST_PKG), main)
-	CI_PKG_TO_TEST=$(shell go list ${DEFAULT_PKG_TO_TEST} | grep -v "server/datastore/mysql" | grep -v "cmd/fleetctl" | grep -v "server/vulnerabilities" | sed -e 's|github.com/fleetdm/fleet/v4/||g')
-else ifeq ($(CI_TEST_PKG), integration)
-	CI_PKG_TO_TEST="server/service"
-else ifeq ($(CI_TEST_PKG), mysql)
-	CI_PKG_TO_TEST="server/datastore/mysql/..."
-else ifeq ($(CI_TEST_PKG), fleetctl)
-	CI_PKG_TO_TEST="cmd/fleetctl/..."
-else ifeq ($(CI_TEST_PKG), vuln)
-	CI_PKG_TO_TEST="server/vulnerabilities/..."
-else
-	CI_PKG_TO_TEST=$(DEFAULT_PKG_TO_TEST)
-endif
-
-ci-pkg-list:
-	@echo $(CI_PKG_TO_TEST)
```

Comment:
@dantecatalfamo I think this was just for debugging?

Reply:
Yeah